utils/mkhtml.py

```diff
@@ -48,7 +48,15 @@
     import grass.script as gs
 except ImportError:
     # During compilation GRASS GIS
-    gs = None
+    _ = str
+
+    class gs:
+        def warning(message):
+            pass
+
+        def fatal(message):
+            pass
```

Comment:
This is confusing to me. Why this is not verbose message in the first place? This is meant primary for compilation anyway, no? When you don't want the things to print `ERROR:...` and exit or print `WARNING:...`, then don't call gs.warning and gs.fatal. When reading the other part of the code, I will not remember that in this file the fatal function is redefined to do nothing.

Reply:
My original intention was to display these warning / fatal messages during the addon installation via g.extension, (then the remote GitHub API server is called) and not to display these messages during compilation.

Reply:
`download_git_commit()` function is called: 

1. During GRASS GIS compilation, when Git repo doesn't exists
2. Addon installation (via g.extension module)